go/opentelemetry.go

```diff
@@ -26,7 +26,6 @@
 package glide
 
 /*
-#cgo LDFLAGS: -lglide_ffi
```

Comment:
oh rly?
We probably don't need `#include <stdlib.h>` too

Reply:
I think you need the stdlib.h. If you include the flag, you will get a compiler warning of a duplicate flag.  I'll test on whether we can remove the stdlib.

Reply:
After testing, the only line that is required is `#include "lib.h"`.